app/src/main/java/app/nexd/android/ui/helper/detail/HelperDetailViewModel.kt

```diff
@@ -57,6 +66,18 @@ class HelperDetailViewModel(private val api: Api) : ViewModel() {
             .blockingSubscribe()
     }
 
+    private fun declineRequest(requestId: Long) {
+        val helpListIdFromRequest =
+            api.helpRequestsControllerGetSingleRequest(requestId).blockingFirst().helpListId
+        helpListIdFromRequest?.let { helpListId ->
```

Comment:
please dont use blockingFirst because it is blocking the thread and does not handle errors properly. Use a flatmap instead

Reply:
fixed with recent push